README.md

```diff
@@ -192,13 +195,21 @@ The backtrace-unity library includes support for capturing Android NDK crashes a
 
 `system.memory` usage related information including memfree, swapfree, and vmalloc.used is now available. Additional VM details and voluntary / nonvountary ctxt switches are included.
 
-## ANRs and Hangs
+## ANRs and Hangs <a name="anr-reporting"></a>
+
+When configuring the backtrace-unity client for an Android deployment, programmers will have a toggle available in the backtrace-unity GUI in the Unity Editor to enable or disable ANR or Hang reports. This will use the default of 5 seconds. This will be made configurable in a future release. 
+
+The `error.type` for these reports will be `Hang`.
+
+## (Early Access) Low Memory Reports <a name="oom-reporting"></a>
+
+Backtrace can detect low memory situations for a game running in Unity on Android devices, and attempt to generate an error report with an associated dump object for further investigation. When configuring the backtrace-unity client for an Android deployment, programmers will have a toggle available in the backtrace-unity GUI in the Unity Editor to enable or disable sending Low memory warnings to Backtrace. 
```

Comment:
` When configuring the backtrace-unity client for an Android deployment,` this feature is available also on iOS.

on Android when we receive a low memory warning, our native android library will set an attribute memory.warning to true and memory.warning.date to current date (with time). With that being said, when crash occurred because of our of memory - for example bad allocaiton, user can recognize it by checking those two attributes. If memory.warning is true and memory.warning.date is close to dump creation date then we're sure that this is a dump generated by our native integration


on iOS - we're monitoring application state and reason why application exited - by comparing for example application.verison, system.version etc. When system.version, application.version are the same, debugger wasn't connected and low memory warning information are available then we're reporting low memory warning.

Reply:
Thanks for specifying the attributes we set (memory.warning and memory.warning.date) in Android. 

Not sure if text adds anything meaningful for iOS, so I wouldn't update anything there unless i'm missing something.